hugo.toml

```diff
@@ -1,5 +1,5 @@
 baseURL = "/"
-title = "SOPS: Secrets OPerationS"
+title = "SOPS: Secrets Operations"
```

Comment:
I think they intentionally capitalize it this way.

Reply:
For some reason, it looks like a typo to me though.
@hiddeco @sabre1041 can you please confirm the appropriate title?

Reply:
Oh...just figured this was done on purpose.